index.html

```diff
@@ -1,389 +1,462 @@
+          </a>
+        </div>
+      </div>
+    </section>
+    <section class="py-8 pb-5 font-work-sans container mx-auto" id="chapters">
+      <h2 class="font-work-sans text-center font-semibold text-3xl">
+        JSLovers's Chapters
+      </h2>
 
-            <div class="flex justify-center items-center mt-8 gap-8">
-                <a
-                    href="https://www.meetup.com/jslovers"
-                    target="_blank"
-                    class="inline-block text-center"
-                >
-                    <img
-                        src="resources/assets/india-gate.png"
-                        alt="delhi jslovers"
-                        class="hover:opacity-50 w-[80px]"
-                    />
-                    <p class="hover:no-underline underline mt-2" >Delhi/NCR</p>
-                </a>
-                <a href="" class="inline-block text-center">
-                    <img
-                        src="resources/assets/bangalore.png"
-                        alt="Bengaluru jslovers"
-                        class="hover:opacity-50 w-[80px]"
-                    />
-                    <p class="hover:no-underline underline mt-2">Bengaluru</p>
-                </a>
-                <a href="" class="inline-block text-center">
-                    <img
-                        src="resources/assets/mumbai.png"
-                        alt="Mumbai JSLovers"
-                        class="hover:opacity-50 w-[80px]"
-                    />
-                    <p class="hover:no-underline underline mt-2">Mumbai</p>
-                </a>
-                <a href="" class="inline-block text-center">
-                    <img
-                        src="resources/assets/bulding.png"
-                        alt="Pune JSLovers"
-                        class="hover:opacity-50 w-[80px]"
-                    />
-                    <p class="hover:no-underline underline mt-2">Pune</p>
-                </a>
-                <a href="" class="inline-block text-center">
-                    <img
-                        src="resources/assets/big-ben.png"
-                        alt="Pune JSLovers"
-                        class="hover:opacity-50 w-[80px]"
-                    />
-                    <p class="hover:no-underline underline mt-2">London</p>
-                </a>
-            </div>
-        </section>
-        <section class="bg-[#F3F6E6] py-8" id="sponsors">
-            <h2 class="font-work-sans font-semibold pl-8 text-2xl text-center">
-                Our Sponsors
-            </h2>
-            <p class="text-l text-center">
-                We are looking for venues, goodies, F&B, and speakers
-                sponsors.<br />
-                If you would like to support us please drop a message to
-                <a href="mailto:neha@jslovers.com" class="underline"
-                    >neha@jslovers.com</a
-                >
-            </p>
+      <div
```

Comment:
On a few resolution, the last logo is not aligned to left or right. It’s stuck in center of the grid